bigchaindb/common/transaction.py

```diff
@@ -677,28 +762,43 @@ def create(cls, owners_before, owners_after, metadata=None, asset=None,
             raise TypeError('`owners_after` must be a list instance')
 
         metadata = Metadata(metadata)
+
+        ffils = []
+        conds = []
+
+        # generate_conditions
+        for owner_after in owners_after:
+            # TODO: Check types so this doesn't fail unpacking
```

Comment:
I think you can remove this, we don't need type checking everywhere for every value.